numba/errors.py

```diff
@@ -241,6 +241,35 @@ def termcolor():
 https://github.com/numba/numba/issues/new
 """
 
+interpreter_error_info = """
+Unsupported Python functionality was found in the code Numba was trying to
+compile. This error could be due to invalid code, does the code work
+without Numba? (A quick way to check is to temporarily disable Numba JIT by
```

Comment:
Can also shorten this to "(To temporarily disable the Numba JIT, set ... and then rerun the code.)"